README.md

```diff
@@ -1,16 +1,15 @@
 # HelloID-Conn-Prov-Notification-Topdesk
 
-| :warning: Warning                                                                                                                                                                                                                                 |
-| :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| Please be aware that the notifications only can be triggered by [events](https://docs.helloid.com/en/provisioning/notifications--provisioning-/notification-events--provisioning-.html). If you need entitlements please use the Target connector [HelloID Topdesk target system](https://github.com/Tools4everBV/HelloID-Conn-Prov-Target-Topdesk) |
+> [!IMPORTANT]
+> Please be aware that the notifications only can be triggered by [events](https://docs.helloid.com/en/provisioning/notifications--provisioning-/notification-events--provisioning-.html). If you need entitlements please use the Target connector [HelloID Topdesk target system](https://github.com/Tools4everBV/HelloID-Conn-Prov-Target-Topdesk)
 
 
-| :information_source: Information                                                                                                                                                                                                                                                                                                                                                       |
-| :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| This repository contains the connector and configuration code only. The implementer is responsible to acquire the connection details such as username, password, certificate, etc. You might even need to sign a contract or agreement with the supplier before implementing this connector. Please contact the client's application manager to coordinate the connector requirements. |
+> [!IMPORTANT]
+> This repository contains the connector and configuration code only. The implementer is responsible to acquire the connection details such as username, password, certificate, etc. You might even need to sign a contract or agreement with the supplier before implementing this connector. Please contact the client's application manager to coordinate the connector requirements.
+
 <br />
 <p align="center"> 
-  <img src="https://www.tools4ever.nl/connector-logos/topdesk-logo.png">
+  <img src="https://github.com/Tools4everBV/HelloID-Conn-Prov-Notification-Topdesk/blob/main/Logo.png?raw=true">
```

Comment:
Not sure if this is ok.

Reply:
This is the new default method (the logo also is shown when showing the readme in HelloID and we are not dependent on the tools4ever.nl logo availability)